go.mod

```diff
@@ -42,20 +42,20 @@ require (
 	github.com/rancher-sandbox/ele-testhelpers v0.0.0-20221213084338-a8ffdd2b87e3
 	github.com/rancher-sandbox/go-tpm v0.0.0-20220823075603-d273b298fcda
 	github.com/rancher/fleet/pkg/apis v0.0.0-20220318160658-4dc66c946ca2
-	github.com/rancher/lasso v0.0.0-20210709145333-6c6cd7fd6607
+	github.com/rancher/lasso v0.0.0-20220628160937-749b3397db38
 	github.com/rancher/rancher/pkg/apis v0.0.0-20211013185633-a636bda2a00e
 	github.com/rancher/steve v0.0.0-20220503004032-53511a06ff37
-	github.com/rancher/system-agent v0.2.8
+	github.com/rancher/system-agent v0.2.14-0.20230106171935-24c523a4408a
```

Comment:
FYI I used the commit-sha since there has not been a tagged release, maybe we should wait for the real release?

Reply:
Is this the `system-agent` version used in 2.7.2 ?!

Reply:
Probably not, just a placeholder for now..

Reply:
Looking at the system-agent repo it seems the next release is scheduled for 2.7.2, so I would merge this one and then update to a tagged release when available.

Reply:
We should open an issue or similar to not forget though